lib/agent.js

```diff
@@ -5,7 +5,7 @@ var parseUrl = require('url').parse
 var path = require('path')
 
 var afterAll = require('after-all-results')
-var debug = require('debug')('elastic-apm')
+var ConsoleLogLevel = require('console-log-level')
```

Comment:
It's kind of a convention that variables that start with an uppercase letter like this needs to be instantiated with the `new` keyword, so I'd probably change it to `consoleLogLevel`

Reply:
fixed